### PR TITLE
Throw an error if a scripts URL is specified but no scripts can be downloaded from it

### PR DESCRIPTION
**Pull request by csrwng**

### CODE REVIEW

**Comment by csrwng** [comment]
@bparees ptal

File: pkg/scripts/install.go [hunk 1]
```diff
@@ -29,29 +29,41 @@ type ScriptHandler interface {
...
 }
 
-const sourcesRootAbbrev = "<source-dir>/"
+const (
+	sourcesRootAbbrev = "<source-dir>/"
+
+	// ScriptURLHandler is the name of the script URL handler
+	ScriptURLHandler = "script URL handler"
+
+	// ScriptURLHandler is the name of the image URL handler
```
Comment (bparees) [inline pkg/scripts/install.go]: ImageURLHandler

**Comment by bparees** [comment]
one nit and squash, then lgtm

**Comment by csrwng** [comment]
thx, [merge]

**Comment by csrwng** [comment]
fixed gofmt error

**Pull request merged**